airflow/cli/commands/task_command.py

```diff
@@ -182,9 +184,25 @@ def task_run(args, dag=None):
     if args.interactive:
         _run_task_by_selected_method(args, dag, ti)
     else:
+        airflow_logger = None
+        airflow_logger_handlers = []
+        if CAN_FORK:
+            airflow_logger_handlers = logging.getLogger('airflow.task').handlers
+            airflow_logger = logging.getLogger('airflow')
+            for handler in airflow_logger_handlers:
+                airflow_logger.addHandler(handler)
+            airflow_logger.setLevel(logging.getLogger('airflow.task').level)
+            airflow_logger.propagate = False
```

Comment:
Is this the best place to put this? Could it perhaps instead be put in StandardTaskRUnner._start_by_fork?

This all feels a little odd and out of place here. At the very least this needs loads of explanatory comments!

What does this change do if someone runs `airflow task run` directly?